### DISCUSSION TITLE
Add resource attributes to prometheus exporter?

### DISCUSSION THREAD

**Discussion by dominikjeske** [open-telemetry/opentelemetry-dotnet#5511] [General]
Documentations is mentioning about "A Prometheus Exporter MAY offer configuration to add resource attributes as metric attributes" -> https://opentelemetry.io/docs/specs/otel/metrics/sdk_exporters/prometheus/. Is this option supported in .NET?

**Comment by vishweshbankwar** [comment]
@dominikjeske - Prometheus added capability to export Resource attributes as target_info. https://github.com/open-telemetry/opentelemetry-dotnet/pull/5407

It is available in 1.8.0-rc.1

If you are looking for https://github.com/open-telemetry/opentelemetry-dotnet/pull/5489. It is still under discussions.

**Reply by dominikjeske** [reply]
Thanks for info
